static/app/components/button.tsx

```diff
@@ -129,7 +131,7 @@ class BaseButton extends React.Component<ButtonProps, {}> {
     // Doing this instead of using `Tooltip`'s `disabled` prop so that we can minimize snapshot nesting
     if (title) {
       return (
-        <Tooltip skipWrapper={!disabled} {...tooltipProps} title={title}>
```

Comment:
can we keep this, i think tooltips don't work on disabled buttons otherwise because of mouse events

Reply:
@scttcper yes for sure, this branch is more of a proof of concept and we should pick whatever is useful from here. There’s a solid chance I went a bit overboard with some changes…